example/synopsis.pl

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+
+use Term::EditLine qw(CC_EOF);
+
+use v5.14;
+
+my $el = Term::EditLine->new('progname');
+$el->set_prompt ('# ');
+
+$el->add_fun ('bye','desc',sub { say "\nbye"; return CC_EOF; });
+
+$el->parse('bind','-e');
+$el->parse('bind','^D','bye');
+
+while (defined($_ = $el->gets())) {
```

Comment:
nit: I know that this is from the original POD (which I didn't write), but while we are in here, can we change this here (and in the POD) to:

```perl
while (defined( my $line = $el->gets )) {
...
```

this code has unnecessary parans and I am not a fan of using the topic variable like this in blocks of more than one line.